versions.tf

```
@@ -3,11 +3,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "~> 3.0"
-    }
-    null = {
-      source  = "hashicorp/null"
-      version = ">= 3.0.0"
+      version = "~> 3.22.0"
```

```suggestion
      version = "~> 3.22"
```

to allow upper versions

---

done!